tdrs-frontend/src/actions/requestAccess.test.js

```diff
@@ -0,0 +1,77 @@
+import axios from 'axios'
+import thunk from 'redux-thunk'
+import configureStore from 'redux-mock-store'
+
+import {
+  requestAccess,
+  PATCH_REQUEST_ACCESS,
+  SET_REQUEST_ACCESS,
+  SET_REQUEST_ACCESS_ERROR,
+  CLEAR_REQUEST_ACCESS,
+} from './requestAccess'
+
+describe('actions/requestAccess.js', () => {
+  const mockStore = configureStore([thunk])
+
+  it('sends a PATCH request when requestAccess is called', async () => {
```

Comment:
Update the frontend to reflect the different REST method for the set_profile end point